fuel-vm/tests/metadata.rs

```diff
@@ -196,14 +198,16 @@ fn get_transaction_fields() {
 
     let tx = TransactionBuilder::create(contract, salt, storage_slots)
         .add_output(Output::contract_created(contract_id, state_root))
-        .finalize_checked(height, &params);
+        .finalize_checked(height, &params, client.gas_costs());
 
     client.deploy(tx);
 
     let mut predicate = vec![0u8; 128];
     let mut predicate_data = vec![0u8; 512];
 
     rng.fill(predicate.as_mut_slice());
+    // make predicate valid by replacing first bytes with RET(1)
```

Comment:
Hmm, it looks weird because we filled the predicate with random data, but after we replaced the first command with the `Opcode::RET(0x01)` -> all random data is useless. Maybe we don't need to sue random bytes at all?

Reply:
This test is mainly about testing offsets, and not about checking predicate validity. Now that predicate verification is part of IntoChecked, we can't test using invalid predicates anymore.

Reply:
I get it, but we can replace 100 random bytes with one opcode - `Opcode::RET(0x01)`. Because we already introduced a valid predicate